### PR TITLE
feat(runtime): update to freedesktop 22.08

### PR DESCRIPTION
**Pull request by Lctrs**

### CODE REVIEW

File: com.valvesoftware.Steam.CompatibilityTool.Proton.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+runtime-version: beta
+sdk: org.freedesktop.Sdk//22.08beta
 build-extension: true
 appstream-compose: false
 
 sdk-extensions:
   - org.freedesktop.Sdk.Compat.i386
   - org.freedesktop.Sdk.Extension.toolchain-i386
-  - org.freedesktop.Sdk.Extension.mingw-w64
+  - org.freedesktop.Sdk.Extension.mingw-w64//beta
```
Comment (gasinvein) [inline com.valvesoftware.Steam.CompatibilityTool.Proton.yml:12]: You don't need to specify sdk extension branch here. Besides, `beta` is wrong, it's `22.08beta` (i.e. matches the sdk).

Comment (Lctrs) [reply com.valvesoftware.Steam.CompatibilityTool.Proton.yml]: Anyway, specifying the branch seems to not be supported by flatpak-builder.

**Comment by Lctrs** [comment]
bot, build

**Comment by Lctrs** [comment]
bot, build

**Pull request merged**